bin/cohere-embed-v3-services.ts

```diff
@@ -33,12 +34,13 @@ const cohereEmbedModel = process.env.COHERE_EMBED_MODEL!;
 const imageVersion = process.env.IMAGE_VERSION!;
 const ecrRepositoryName = process.env.ECR_REPOSITORY_NAME!;
 const dockerfileName = process.env.DOCKERFILE_NAME!;
-const port = process.env.PORT!;
+const cdkDeployPort = process.env.PORT!;
```

Comment:
**Suggestion:** The `cdkDeployPort` variable is being assigned from `process.env.PORT`, but the variable name `PORT` is not descriptive. Consider renaming it to something more specific like `CDK_DEPLOY_PORT`.
> 📚  **Medium Priority** 
 > [best practice, importance: 6]
```suggestion
const cdkDeployPort = process.env.CDK_DEPLOY_PORT!;
```